user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_ElementFieldDescriptor.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: DOM_ElementFieldDescriptor
+---
+
+# ElementFieldDescriptor
+
+- **FieldValue type**: `GUID` of an `Element`
+- **Multiple values supported**: :heavy_check_mark:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| Element | string | string |
+| Element with multiple values enabled| List\<string\> | string (ListValueWrapper) |
+
+Available from DataMiner 10.1.10/10.2.0 onwards. Can be used to define that a field should contain the `ID` of an `Element`. The `ID` must be saved as a string according to the common [DMA ID]/[ELEMENT ID] format (e.g. "868/65874"). There is a `ViewIds` list property that can be used to define whether the `Elements` should be in any of these `Views`. The selected `Element` is saved as a string.
```

Comment:
'The selected `Element` is saved as a string: I assume we save the `ElementID` as a string?